tests/test_webhook.py

```diff
@@ -23,6 +23,12 @@
 # The key is being used for testing and computing the signature.
 key = app.config['HOOK_SECRET_KEY']
 
+# a fake type-media issues list for testing
```

Comment:
Do you mean a comment explaining the 3 fields?
`'issue_number, error_value, domain'` it replicates the file structure once read for testing (but would/should probably change to be something slightly better).